os-ui/src/pages/CourseModulePage/CourseModulePage.tsx

```diff
@@ -13,7 +13,7 @@ import DiscussionForum from './DiscussionPage/DiscussionForum';
 
 const CourseModulePage = ({ userInfo }:{userInfo:any}) => {
   const [isDiscBtnpressed, setDiscBtnpressed] = useState<boolean>(
-    localStorage.getItem('isDiscBtnpressed') === 'true', // Checking the value in localStorage and transforming it into boolean
```

Comment:
Please don't use localStorage and sessionStorage as we have Redux store in our application